modules/chroma/chroma_test.go

```diff
@@ -23,33 +23,23 @@ func TestChroma(t *testing.T) {
 		// restEndpoint {
 		restEndpoint, err := ctr.RESTEndpoint(ctx)
 		// }
-		if err != nil {
-			tt.Fatalf("failed to get REST endpoint: %s", err)
-		}
+		require.NoErrorf(tt, err, "failed to get REST endpoint")
 
 		cli := &http.Client{}
 		resp, err := cli.Get(restEndpoint + "/docs")
-		if err != nil {
-			tt.Fatalf("failed to perform GET request: %s", err)
-		}
+		require.NoErrorf(tt, err, "failed to perform GET request")
 		defer resp.Body.Close()
 
-		if resp.StatusCode != http.StatusOK {
-			tt.Fatalf("unexpected status code: %d", resp.StatusCode)
-		}
+		require.Equalf(tt, http.StatusOK, resp.StatusCode, "unexpected status code: %d", resp.StatusCode)
```

Comment:
This is an example of a message that people might think is helpful but isn't in my mind as the values are output anyway and is obvious from the code its a status check.